### ISSUE TITLE
new props don't cause Presenters to re-render

### ISSUE THREAD

**Issue by greypants** [vigetlabs/microcosm#260]
**This issue popped up when upgrading from `11.6.0` to `12.2.0`**

I noticed this issue when components stopped swapping out on route changes. Simplified, the issue looks like this:

```js
class Application extends Presenter {
  render() {
    <div className="some-layout">
      { this.props.children }
    </div>
  }
}
```

Routes:
```js
<Route path="/" component={Application}>
  <Route path="foo" component={Foo} />
  <Route path="bar" component={Bar} />
</Route>
```

Changing from `/foo` to `/bar`, does not cause a re-render, and the component is not swapped out.

Simply adding `children` to the model seems to force the Presenter to update when the prop changes, and works around the issue.

```js
class Application extends Presenter {
  getModel (props) {
    return {
      children: () => props.children
    }
  },

  render () {
    <div className="some-layout">
      { this.props.children }
    </div>
  }
}
```

But as I understand it, prop changes should cause `render` to be called again.

**Comment by nhunzaker** [comment]
This should be fixed in 12.2.1, currently in release candidate. Will close after we publish to stable.

**Comment by nhunzaker** [comment]
Fixed: https://github.com/vigetlabs/microcosm/releases/tag/v12.2.1
